rest/model/codegen_test.go

```diff
@@ -62,3 +52,17 @@ func TestWords(t *testing.T) {
 	s := "thisIsAFieldName"
 	assert.Equal(t, words(s), []string{"this", "is", "a", "field", "name"})
 }
+
+func TestGqlTypeToGoType(t *testing.T) {
+	cases := map[string]string{
+		"String":  "string",
+		"String!": "string",
+		"Map":     "map[string]interface{}",
+		"[Int!]!": "[]int",
+	}
+	for input, output := range cases {
+		assert.Equalf(t, output, gqlTypeToGoType(input, false), "case: %s; expected: %s", input, output)
```

Comment:
It's not an issue but the format string seems unnecessary: doesn't it already print something like that?